content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -56,6 +56,13 @@ the sample root certificates as the intermediate certificate.
         --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
+1. Update Helm’s dependencies:
+
+    {{< text bash >}}
+    $ helm repo add istio.io "https://storage.googleapis.com/istio-prerelease/daily-build/master-latest-daily/charts"
```

Comment:
@sdake: do you know what is (or will be) the "proper way" of doing this when using an official Istio release, instead of a daily build?